src/main/resources/META-INF/plugin.xml

```diff
@@ -60,7 +60,7 @@
   <depends>com.intellij.modules.python</depends>
   <depends>org.jetbrains.plugins.terminal</depends>
 
-  <idea-version since-build="202" until-build="203.*"/>
+  <idea-version since-build="202" until-build="211.*"/>
```

Comment:
It won't compile on 211. There have been incompatible API changes. Please check them on the latest EAP.

Reply:
It will compile actually, but with deprecation warnings. BTW I've fixed a build issue by updating dependencies and I've updated to until-build="211.*" in master. Could you please rebase your PR?

Reply:
I merged in code from your master branch, should be enough to resolve that.